schema/bitbucket_server.schema.json

```
@@ -34,7 +34,7 @@
           "default": true
         },
         "requestsPerHour": {
-          "description": "Requests per hour permitted. This is an average, calculated per second.",
+          "description": "Requests per hour permitted. This is an average, calculated per second. Internally the burst limit is set to 500, which implies that for a requests per hour limit as low as 1, users will continue to be able to send a maximum of 500 requests immediately, provided that the complexity cost of each request is 1.",
```

```suggestion
          "description": "Requests per hour permitted. This is an average, calculated per second. Internally, the burst limit is set to 500, which implies that for requests per hour limit is as low as 1, users will continue to be able to send a maximum of 500 requests immediately, provided that the complexity cost of each request is 1.",
```

Similar on other files.